docs/installation/linux_and_mac.rst

```diff
@@ -0,0 +1,149 @@
+
+1. Download and Run the Installation Script
+-------------------------------------------
+
+For systems with `wget` (e.g. Debian derivatives like Ubuntu):
+
+.. parsed-literal::
+
+      $ wget \https://raw.githubusercontent.com/tethysplatform/tethys/|branch|/scripts/install_tethys.sh
+      $ . install_tethys.sh
+
+For Systems with `curl` (e.g. Mac OSX, and RedHat derivatives like CentOS or Fedora):
+
+.. parsed-literal::
+
+      $ curl \https://raw.githubusercontent.com/tethysplatform/tethys/|branch|/scripts/install_tethys.sh -o ./install_tethys.sh
+      $ . install_tethys.sh
+
+.. note::
+
+    You can customize your tethys installation by passing command line options to the installation script. The available options can be listed by running::
+
+         $ . install_tethys.sh --help
+
+    Each option is also descriped here:
+
+        * `-t, --tethys-home <PATH>`:
+                path for tethys home directory. Default is /usr/lib/tethys on Linux and /Library/Applicaiton/tethys on Mac OS.
+        * `-a, --allowed-host <HOST>`:
+                hostname or IP address on which to server tethys. Default is 127.0.0.1.
+        * `-p, --port <PORT>`:
+                port on which to server tethys. Default is 8000.
+        * `-b, --branch <BRANCH_NAME>`:
+                repository branch to checkout. Default is 'dev'.
+        * `-c, --conda-home <PATH>`:
+                path to conda home directory. Default is ${TETHYS_HOME}/miniconda.
+        * `--db-username <USERNAME>`:
+                username that the tethys database server will use. Default is 'tethys_default'.
+        * `--db-password <PASSWORD>`:
+                password that the tethys database server will use. Default is 'pass'.
+        * `--db-port <PORT>`:
+                port that the tethys database server will use. Default is 5436.
+        * `-S, --superuser <USERNAME>`:
+                Tethys super user name. Default is 'admin'.
+        * `-E, --superuser-email <EMAIL>`:
+                Tethys super user email. Default is ''.
+        * `-P, --superuser-pass <PASSWORD>`:
+                Tethys super user password. Default is 'pass'.
+        * `--install-docker`:
+                Flag to include Docker installation as part of the install script (Linux only). See `2. Install Docker (OPTIONAL)`_ for more details.
+        * `--docker-options <OPTIONS>`:
+                Command line options to pass to the `tethys docker init` call if --install-docker is used. Default is "'-d'".
+        * `-x`:
+                Flag to turn on shell command echoing.
+        * `-h, --help`:
+                Print this help information.
+
+    Here is an example of calling the installation script with customized options::
+
+        $ . install_tethys.sh -t ~/tethys -a localhost -p 8005 -c ~/tethys/conda --db-username tethys_db_user --db-password db_user_pass --db-port 5437 -S tethys -E email@example.com -P tpass
```

Comment:
I love how fully featured this is! I'm also glad that there are defaults for all of these, because that is a lot of options.